implementation/.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
Shouldn't need sub directory .gitignore files

Reply:
Agreed.  Will change it.